cmd/cli/comm_trans.go

```diff
@@ -97,6 +97,9 @@ func (c *CommTrans) GenPreExeRes(ctx context.Context) (
 		if tmpReq != nil {
 			preExeReqs = append(preExeReqs, tmpReq)
 		}
+		if tmpReq == nil && err == nil {
```

Comment:
use "else if" may reduce a tmpReq value comparing?

Reply:
Fix it